lib/index.ts

```diff
@@ -158,15 +153,16 @@ class PWMetrics {
     }
   }
 
-  async launchChrome(): Promise<ChromeLauncher> {
+  async launchChrome(): Promise<LaunchedChrome|Error> {
     try {
-      this.launcher = new ChromeLauncher();
-      await this.launcher.isDebuggerReady();
-      return this.launcher;
-    } catch(e) {
       console.log(messages.getMessage('LAUNCHING_CHROME'));
-      await this.launcher.run();
-      return this.launcher;
+      return this.launcher = await launch({
+        port: this.flags.port,
```

Comment:
Yes we missed that, I added the port to lighthouse. I think it's best let chrome-launcher find an available port and use that to prevent port-in-use errors. @samccone Lighthouse is port-agnostic too right? It should work on any available port correct?